cmd/plume/release.go

```diff
@@ -445,3 +469,72 @@ func doAWS(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 		}
 	}
 }
+
+func doS3(spec *fcosChannelSpec) {
+	api, err := aws.New(&aws.Options{
+		CredentialsFile: awsCredentialsFile,
+		Profile:         spec.Profile,
+		Region:          spec.Region,
+	})
+	if err != nil {
+		plog.Fatalf("creating aws client: %v", err)
+	}
+
+	err = api.UpdateBucketObjectsACL(spec.Bucket, filepath.Join("prod", "streams", specChannel, "builds", specVersion), specPolicy)
+	if err != nil {
+		plog.Fatalf("updating object ACLs: %v", err)
+	}
+}
+
+func modifyReleaseMetadataIndex(spec *fcosChannelSpec, commitId string) {
+	api, err := aws.New(&aws.Options{
+		CredentialsFile: awsCredentialsFile,
+		Profile:         spec.Profile,
+		Region:          spec.Region,
+	})
+	if err != nil {
+		plog.Fatalf("creating aws client: %v", err)
+	}
+
+	path := filepath.Join("prod", "streams", specChannel, "releases.json")
+
+	f, err := api.DownloadFile(spec.Bucket, path)
+	if err != nil {
+		plog.Fatalf("downloading release metadata index: %v", err)
+	}
+	defer f.Close()
+
+	data, err := ioutil.ReadAll(f)
+	if err != nil {
+		plog.Fatalf("reading release metadata index: %v", err)
+	}
+
+	var m ReleaseMetadata
+	err = json.Unmarshal(data, &m)
+	if err != nil {
+		plog.Fatalf("unmarshaling release metadata json: %v", err)
+	}
+
+	url, err := url.Parse(fmt.Sprintf("https://%v.s3.amazonaws.com/prod/streams/%v/builds/%v/release.json", spec.Bucket, specChannel, specVersion))
+	if err != nil {
+		plog.Fatalf("creating metadata url: %v", err)
+	}
+
+	m.Releases = append(m.Releases, BuildMetadata{
+		CommitHash: specCommitId,
+		Version:    specVersion,
+		Endpoint:   url.String(),
+	})
+
+	m.Timestamp = time.Now().Format("2006-01-02T15:04:05Z")
+
```

Comment:
Missing e.g.:

```go
m.Observation = "Not for direct consumption; do not depend on this file or its contents."
```

:)

Reply:
My thinking was that that line would be inside of the initial document and then is just persisted rather than us overwriting the value each time `plume` is ran.